crazy_miner/telemedicin_models.py

```
@@ -367,17 +374,36 @@ class CrazyMinerPayment(models.Model):
+        
+        with transaction.atomic():
+            self.status = 'completed'
+            self.completed_at = timezone.now()
+            self.save()
+            
+            # شارژ کیف پول کاربر
+            box_money, created = BoxMoney.objects.get_or_create(
+                user=self.user,
+                defaults={'balance': 0}
+            )
+            box_money.balance = F('balance') + self.amount
```

Ruff (0.12.2)

Undefined name `F`

(F821)